Cutting columns option

RomainDereu opened the pull request:
This new option cuts the first column from the csv data. resto-googleforms.csv has the same data as resto.csv but needs to have its first column deleted before being processed. the command for the cutting column option is -I or ignore-first-column

RomainDereu commented:
@adrien-cotte 
Hi there, 
I implemented the column cutting option which can be activated with the option -I from the command line.
I also added a description in the readme file as well as a resto-googleforms.csv file for testing purposes. Note that the content is the same as resto.csv with random data inserted in the first column.

Hope this corresponds to what you had in mind,

Cheers

adrien-cotte commented:
Thank you for the contribution, a good start.

Can you add default value `False` to `ignore_first_column` in function `read_and_aggregate_csv`?
A new option like that should not "break" the ancient way to call `read_and_aggregate_csv`, look at https://github.com/adrien-cotte/majority-judgment/blob/main/discord-bot/major_bot.py#L260

Moreover, the option should be `-I, --ignore-first-column` instead of `-I {True,False}, --ignore-first-column {True,False}`, don't you think :)?

RomainDereu commented:
@adrien-cotte 
Thanks a lot for the review Adrien!

> Can you add default value False to ignore_first_column in function read_and_aggregate_csv?
My bad, I failed to analyze the bot part of your application.
Not sure if needed but I added the default option as requested and also changed major_bot.py. 
Let me know if this isn't necessary.
dc676cb4687cc4fbd054487378fa9a8e570de41e

> Moreover, the option should be -I, --ignore-first-column instead of -I {True,False}, --ignore-first-column {True,False}, don't you think :)?

Thanks, I copied the lang option but it makes way more sense the way you pointed out.
I made the change here: b33d1c52fb22c20824a8abbd0a1b56e45967dc02
